workflows/e2e/action.yaml

```diff
@@ -0,0 +1,105 @@
+    default: "greenhouse"
+  kubernetes-version:
+    description: "The Kubernetes version used to spin up clusters"
+    required: false
+    default: "v1.30.4"
+  go-version:
+    description: "The Go version to build greenhousectl"
+    required: false
+    default: "1.23"
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout tools repo
+      uses: actions/checkout@v4
+      with:
+        repository: cloudoperators/greenhouse
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ inputs.go-version }}
+    - name: Build greenhousectl
+      shell: bash
+      run: |
+        make build-greenhousectl
+    - name: Create k8s Kind Cluster
+      uses: helm/kind-action@v1.10.0
+      with:
+        install_only: true
+    - name: compose config.json
+      shell: bash
+      run: |
+        cat << EOF > $RUNNER_TEMP/config.json
+        {
+            "config":[
+              {
+                  "cluster":{
+                    "name": "${{ inputs.remote-cluster-name }}",
+                    "version": "${{ inputs.kubernetes-version }}"
+                  }
+              },
+              {
+                  "cluster":{
+                    "name": "${{ inputs.admin-cluster-name }}",
+                    "namespace": "${{ inputs.namespace }}",
+                    "version": "${{ inputs.kubernetes-version }}"
+                  },
+                  "dependencies":[
+                    {
+                        "manifest":{
+                          "release":"greenhouse",
+                          "chartPath":"charts/idproxy",
+                          "crdOnly":true
+                        }
+                    },
+                    {
+                        "manifest":{
+                          "release":"greenhouse",
+                          "chartPath":"charts/manager",
+                          "valuesPath":"dev-env/localenv/sample.values.yaml",
+                          "crdOnly":false,
+                          "webhook":{
+                              "devMode":false,
+                              "dockerFile":"./",
+                              "envs":[
+                                {
+                                    "name":"CONTROLLERS_ONLY",
+                                    "value":"true"
+                                }
+                              ]
+                          }
+                        }
+                    }
+                  ]
+              }
+            ]
+        }
+        EOF
+    - name: run greenhousectl
+      shell: bash
+      run: |
+        bin/greenhousectl dev setup -f $RUNNER_TEMP/config.json
+    - name: export kubeconfig
+      shell: bash
+      run: |
+        kind export kubeconfig --name ${{ inputs.admin-cluster-name }} --kubeconfig $RUNNER_TEMP/e2e.kubeconfig
+        kind export kubeconfig --internal --name ${{ inputs.admin-cluster-name }} --kubeconfig $RUNNER_TEMP/e2e.internal.kubeconfig
```

Comment:
Will the second line not override the exported kubeconfig from the first line?

Reply:
No it would not, because the first one exports to `--kubeconfig $RUNNER_TEMP/e2e.kubeconfig` and in the second there is an additional **internal** in the file name